### ISSUE TITLE
cobalt-0.0.5.pom xml error for Gradle but not for Maven

### ISSUE THREAD

**Issue by zapisator** [Auties00/Cobalt#495]
This

```bash
This page contains the following errors:
error on line 1 at column 7: XML declaration allowed only at the start of the document
Below is a rendering of the page up to the first error.
```
what one can see at https://repo1.maven.org/maven2/com/github/auties00/cobalt/0.0.5/cobalt-0.0.5.pom

Something like that one can see attempting to build on Gradle as part of Java project, or part of Android project. For example in Android mobile app:
```html
> Task :prepareKotlinBuildScriptModel UP-TO-DATE
[Fatal Error] cobalt-0.0.5.pom:3:1: Content is not allowed in prolog.
[Fatal Error] cobalt-0.0.5.pom:3:1: Content is not allowed in prolog.
[Fatal Error] cobalt-0.0.5.pom:3:1: Content is not allowed in prolog.
[Fatal Error] cobalt-0.0.5.pom:3:1: Content is not allowed in prolog.
[Fatal Error] cobalt-0.0.5.pom:3:1: Content is not allowed in prolog.

BUILD SUCCESSFUL in 1s
```

```xml
 <?xml version="1.0" encoding="UTF-8"?>
<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
    <modelVersion>4.0.0</modelVersion>
    <groupId>com.github.auties00</groupId>
    <artifactId>cobalt</artifactId>
    <version>0.0.5</version>
    <name>${project.groupId}:${project.artifactId}</name>
    <description>Standalone fully-featured Whatsapp Web API for Java and Kotlin</description>
    <url>https://github.com/Auties00/Cobalt</url>

    <developers>
        <developer>
            <name>Alessandro Autiero</name>
            <email>alautiero@gmail.com</email>
        </developer>
    </developers>

    <licenses>
...
```

The essence is that there is a space character in the first line of pom. It is incorrect. Can you remove it

**Comment by Darker935** [comment]
Patched on #477, clone this repo and run:
`sh
mvn clean install -DskipTests=true`

**Comment by smartkanak** [comment]
When will this be updated on maven?

**Comment by Darker935** [comment]
> When will this be updated on maven? Would be great to be able to use this with Gradle without any problems

Idk, maybe on 0.0.6

**Comment by Darker935** [comment]
You can use doing what I said and setting `mavenLocal()` on the top of your repos

**Comment by zapisator** [comment]
> Patched on #477, clone this repo and run: `mvn clean install -DskipTests=true`

That works, thanks

**Comment by Auties00** [comment]
fixed in 0.0.6
